README.md

```diff
@@ -75,9 +75,13 @@
       - [`editField(id[, opts])` : [Field](#field)](#editfieldid-opts--fieldfield)
       - [`deleteField(id)` : void](#deletefieldid--void)
       - [`changeFieldId (currentId, newId)` : void](#changefieldid-currentid-newid--void)
-      - [`changeEditorInterface (fieldId, widgetId[, settings])` : void](#changeeditorinterface-fieldid-widgetid-settings--void)
-      - [`resetEditorInterface (fieldId)` : void](#reseteditorinterface-fieldid--void)
-      - [`copyEditorInterface (sourceFieldId, destinationFieldId)` : void](#copyeditorinterface-sourcefieldid-destinationfieldid--void)
+      - [`changeFieldControl (fieldId, widgetId[, settings, widgetNamespace])` : void](#changefieldcontrol-fieldid-widgetid-settings-widgetnamespace--void)
+      - [`resetFieldControl (fieldId)` : void](#resetfieldcontrol-fieldid--void)
+      - [`copyFieldControl (sourceFieldId, destinationFieldId)` : void](#copyfieldcontrol-sourcefieldid-destinationfieldid--void)
+      - [`addSidebarWidget (widgetId[, widgetNamespace, insertBeforeWidgetId, settings])` : void](#addsidebarwidget-widgetid-widgetnamespace-insertbeforewidgetid-settings--void)
```

Comment:
The same: `widgetNamespace` in front. Do we put the widget at the end if `insertBeforeWidgetId` is undefined?

Reply:
as outlined in the docs, yes